src/ucp/proto/proto_common.c

```diff
@@ -307,6 +306,18 @@ ucp_proto_common_get_lane_perf(const ucp_proto_common_init_params_t *params,
                            UCT_PERF_ATTR_FIELD_LATENCY;
     perf_attr.operation  = params->send_op;
 
+    if (params->flags & UCP_PROTO_COMMON_INIT_FLAG_REMOTE_ACCESS) {
+        perf_attr.field_mask      |= UCT_PERF_ATTR_FIELD_LOCAL_SYS_DEVICE;
```

Comment:
IMO the local device should be outside of this if. we can pass it even for local access.